e2e/pages/loadPage.po.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
This class should be removed an its methods should be part to commonActions class